backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -144,7 +144,8 @@ def assign_next(self) -> None:
         reviewers = self.assignees.filter(role=ListRole.reviewer).order_by("order")
 
         # All reviewers have reviewed the draft destruction list
-        if self.assignee == reviewers.last().user:
+        last_reviewer = reviewers.last()
+        if last_reviewer and self.assignee == reviewers.last().user:
```

Comment:
This would only fail if no reviewers have been assigned to the list :thinking: Is it needed?

Also:
```suggestion
        if last_reviewer and self.assignee == last_reviewer.user:
```

Reply:
> This would only fail if no reviewers have been assigned to the list 🤔 Is it needed?
> 
> Also:

I needed this fix to make everything work again so I guess so 😅 , and yeas, last_reviewer should be used.